process_cohort.smk

```diff
@@ -74,6 +120,19 @@ else:
 targets = []
 include: 'rules/cohort_common.smk'
 
+# assemble with hifiasm
+include: 'rules/cohort_hifiasm.smk'
+if 'trio_assembly' in config['cohort_targets']:
+    # assembly and stats
+    targets.extend([f"cohorts/{cohort}/hifiasm/{trio}.asm.dip.{infix}.{suffix}"
+                for suffix in ['fasta.gz', 'fasta.stats.txt']
+                for infix in ['hap1.p_ctg', 'hap2.p_ctg']
```

Comment:
Perhaps add the parent ID to the output files too to clarify which parent is hap1 and which parent is hap2

Reply:
Good idea

Reply:
Added a haplotype key file to the output since hifiasm doesn't seem to have an option for haplotype-specific file prefixes